lib/improver/threshold.py

```diff
@@ -307,11 +308,13 @@ def process(self, input_cube):
         # TODO: Correct when formal cf-standards exists
         # Force the metadata to temporary conventions
         if self.below_thresh_ok:
-            cube.attributes.update({'relative_to_threshold': 'below'})
+            find_threshold_coordinate(cube).attributes.update(
+                {'spp__relative_to_threshold': 'below'})
```

Comment:
I think this coordinate attribute should be set in the _add_threshold_coord function. It feels wrong to create this coord and then have to find it again in the same plugin.

Reply:
Yes, I think you're right, so I've amended the plugin, so that the attribute is added to the coordinate when the threshold coordinate is created.